CondCore/BeamSpotPlugins/interface/BeamSpotPayloadInspectorHelper.h

```diff
@@ -681,14 +685,15 @@ namespace simBeamSpotPI {
    */
   template <class PayloadType>
   class SimBSParamsHelper {
-    typedef std::array<double, 9> bshelpdata;
+    typedef std::array<double, 10> bshelpdata;
 
   public:
     SimBSParamsHelper(const std::shared_ptr<PayloadType>& bs) {
       // fill in the values
       m_values[0] = bs->x(), m_values[1] = bs->y(), m_values[2] = bs->z();
       m_values[3] = bs->sigmaZ(), m_values[4] = bs->betaStar(), m_values[5] = bs->emittance();
```

Comment:
```suggestion
      m_values[parameters::X] = bs->x(), m_values[parameters::Y] = bs->y(), m_values[parameters::Z] = bs->z();
      m_values[parameters::sigmaZ] = bs->sigmaZ(), m_values[parameters::betaStar] = bs->betaStar(),
      m_values[parameters::emittance] = bs->emittance();
```

Reply:
thanks, addressed in [9877ac0](https://github.com/cms-sw/cmssw/pull/41967/commits/9877ac09b974681ef5d0bf058326d23e5f0bc8e0)